main.go

```diff
@@ -278,7 +278,7 @@ func initSelfNodeRemediationAgent(mgr manager.Manager) {
 
 	// but the reboot time needs be at least the time we know we need for determining a node issue and trigger the reboot!
 	// 1. time for determine node issue
-	minTimeToAssumeNodeRebooted := (apiCheckInterval + apiServerTimeout) * time.Duration(maxErrorThreshold) + maxTimeForNoPeersResponse
+	minTimeToAssumeNodeRebooted := (apiCheckInterval+apiServerTimeout)*time.Duration(maxErrorThreshold) + maxTimeForNoPeersResponse
```

Comment:
why this change?

Reply:
Not something I've done manually, I assume it's due to update in the linter